Implementation of configurable timeouts and automatic rollback (v2) (#58)
chef/kitchen-vcenter: README.md

The following parameters should be set in the main `driver_config` section as they are common to all platforms:
 - `vcenter_disable_ssl_verify` - Whether or not to disable SSL verification checks. Good when using self signed certificates. Default: false

>  - `vm_wait_timeout` - How many seconds to wait for VM connectivity. Default: 90

Reviewer: ```suggestion
 - `vm_wait_timeout` - Number of seconds to wait for VM connectivity. Default: 90
```

>  - `vm_wait_interval` - Check interval between tries on VM connectivity. Default: 2.0

Reviewer: Is the "Check interval" a length of time? A number of checks?

Author: It's the length of time between tries to check the VM. So basically "Number of seconds between tries"

>  - `vm_rollback` - Whether to roll back (destroy) VMs which failed VM connectivity checking. Default: false

Reviewer: ```suggestion
 - `vm_rollback` - Decision to roll back (destroy) VMs failing the VM connectivity check. Default: false
```

Author: I'd propose "Automatic roll back (destroy) of VMs failing the connectivity check. Default: false"? Decision sounds a bit weird to me. Non-native speaker though ;)
